docs/user-documentation/getting-started-with-islandora-claw.md

```diff
@@ -0,0 +1,57 @@
+# Getting Started with Islandora CLAW
+
+## How to Add a User
+1. Log in under admin credentials.
+2. Click the **Manage** tab from the top toolbar.
+3. Select **People** tab from the resulting menu. 
+ ![Alt text](../assets/people_page.png "People page")
+4. Enter all the required fields, as indicated by red asterisks, in the Add User page.
+5. Click on “Create New Account” button at the bottom of the page.
```

Comment:
My apologies, this came from the Google Doc and I missed them. I should have known to look for them, smart quotes have burned me in the past.